src/main/app/Resources/styles/print.less

```diff
@@ -0,0 +1,100 @@
+@media print {
+    html {
+        height: auto !important;
+        overflow-y: auto;
+    }
+
+    .app-header-container {
+        display: none;
+    }
+
+    .breadcrumb {
+        display: none;
+    }
+
+    .page-actions {
+        display: none;
+    }
+
+    .tag-cell {
+        display: flex;
+        flex-flow: row wrap;
+    }
+
+    .app .app-menu {
+        display: none;
+    }
+
+    .app-content .app-footer-container {
```

Comment:
`.app-content` is useless.

Reply:
If we remove .app the selector does not have enough specificity.